src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  padding: 0;
+  margin: 0;
```

Comment:
```suggestion
```
you don't need to reset it for all the elements 😉

Reply:
ok, thanks